lib/codegen/meta-python/base/instructions.py

```diff
@@ -181,6 +181,10 @@
     ins=(addr, JT),
     is_branch=True, is_indirect_branch=True, is_terminator=True)
 
+debugtrap = Instruction('debugtrap', r"""
+    Encodes an assembly debug trap.
+    """, other_side_effects=True)
```

Comment:
We should set `can_load` and `can_store` on this too, to represent the fact that a user stopping the program at a `debugtrap` can use a debugger to read and write memory.